README.md

```diff
@@ -1,3 +1,9 @@
+# My modification
```

Comment:
OK I think we should add 2 parallel subsections in each section, with the default is downloading from AWS
README
-------
## Plain text

**Bulk PDF conversion for AWS download**
...
**Bulk PDF conversion for Kaggle download**
...
and then

## Cocitation network
** For AWS download**
...
** For Kaggle download **
-----------
what do you think?

Reply:
I think that's a fine idea, just as long as it's clear to the user.

Reply:
ok, see modifed [README](https://github.com/quynhneo/arxiv-public-datasets_for_kaggle/blob/master/README.md)

Reply:
Sorry to be so picky! Now that I'm reading the instructions, can we merge the `convert_directory.py` script into the original `fulltext.py`, and set it up so that if the user provides those arguments explained in the Kaggle section, it performs accordingly? Or, perhaps to fit in with the existing structure, create a new optional entry to the `config.json.example` file which is the directory containing the Kaggle PDFs, and then the tools will function identically?

Reply:
well that's the right thing to do anyway! I committed what I already had, but it did feel patchy to me to have separated instructions for the same tool. OK, I can go in that direction!

Reply:
Hi Colin,
Thanks for chatting the other days. So I have integrated methods for Kaggle set into the existing structure. Except for the download, and an additional option for `bin/fulltext.py`, everything works the same for two downloading methods.
 Check out the new [README](https://github.com/quynhneo/arxiv-public-datasets_for_kaggle/tree/master).